__test__/abstract_account_plugins.test.ts

```diff
@@ -144,7 +146,7 @@ describe('Abstracted Subscription Program', () => {
           .compose()
           .makePayment(
             // Send a payment from the abstracted account to Joe
-            { sender: aliceAbstractedAccount, acctRef: joe },
+            { sender: aliceAbstractedAccount, _acctRef: joe },
```

Comment:
This was a breaking change introduced by the client generator at some point. We should figure out what version we want to use and stick to it.

Reply:
think this is the only thing left holding this up getting merged